ppafm/GridUtils.py

```diff
@@ -371,24 +391,22 @@ def loadVecFieldNpy( fname, FF = None ):
 	Fy = np.load(fname+'_y.npy' )
 	Fz = np.load(fname+'_z.npy' )
 	lvec = np.load(fname+'_vec.npy' )
+	at_array = np.load(fname+'_atoms.npy')
 	FF = packVecGrid( Fx, Fy, Fz, FF )
 	del Fx,Fy,Fz
-	return FF, lvec
+	return FF, lvec, at_array
 
 def saveVecFieldXsf( fname, FF, lvec, head = XSF_HEAD_DEFAULT ):
-	saveXSF(fname+'_x.xsf', FF[:,:,:,0], lvec, head )
-	saveXSF(fname+'_y.xsf', FF[:,:,:,1], lvec, head )
-	saveXSF(fname+'_z.xsf', FF[:,:,:,2], lvec, head )
+	saveXSF(fname+'_x.xsf', FF[:,:,:,0], lvec, head = head )
+	saveXSF(fname+'_y.xsf', FF[:,:,:,1], lvec, head = head )
+	saveXSF(fname+'_z.xsf', FF[:,:,:,2], lvec, head = head )
 
-def saveVecFieldNpy( fname, FF, lvec , head = XSF_HEAD_DEFAULT ):
+def saveVecFieldNpy( fname, FF, lvec , at_array ):
 	np.save(fname+'_x.npy', FF[:,:,:,0] )
 	np.save(fname+'_y.npy', FF[:,:,:,1] )
 	np.save(fname+'_z.npy', FF[:,:,:,2] )
 	np.save(fname+'_vec.npy', lvec )
-	if (head != XSF_HEAD_DEFAULT ):
-		print("saving atoms")
-		tmp0=head[0]; q=np.zeros(len(tmp0));    #head: [e,[x,y,z],lvec]
-		np.save(fname+'_atoms.npy',[tmp0,head[1][0],head[1][1],head[1][2],q]) #atoms: [e, x, y, z, q]
+	np.save(fname+'_atoms.npy',at_array) # at the moment we have not switched npz, I will do it later, maybe #
```

Comment:
I suggest avoiding such kinds of comments. If anything is planned to be done, there should be an issue about that.

```suggestion
	np.save(fname+'_atoms.npy',at_array)
```